index.html

```diff
@@ -8,7 +8,7 @@
     <title>Microverse Candidates Survey Form</title>
 </head>
 <body>
-  <div id="container">
+  <div id="container1">
```

Comment:
- [ Optional ] You should give the reason why you are requesting the changes since the feedback receiver might not be able to understand why you are asking for change.
- [ Optional ] It is also give some examples/sample/ instead of just saying `a variety of names can be used`. i.e, you can give some example names that they can use, in this case.
- [ Optional ] You could also suggest that they use semantic tags such as `section, article, main, header`, etc instead of a generic `<div>` tag because they help for SEO and accessibility.